src/test/fuzz/addrman.cpp

```diff
@@ -26,6 +28,14 @@
 namespace {
 const BasicTestingSetup* g_setup;
 
+static const std::map<uint8_t, uint8_t> NET_LEN_MAP = {
```

Comment:
In 9bcbbc68682c1a92316d6a2dd8142839c58fe247 (fuzz, refactor: hoist net_len_map in addrman fuzzer to constant): Isn't this already a constant? Seems to just be moving the code out of the function where it's used?